### DISCUSSION TITLE
Suggestion for Music Separation: Separate the Backing Vocals from the Vocals

### DISCUSSION THREAD

**Discussion by emanuelpinheiro** [intel/openvino-plugins-ai-audacity#161] [Ideas]
It would be very interesting if OpenVINO separated the Backing Vocals from the vocals, it would help a lot for those who just want solo vocals or vice versa.

**Comment by RyanMetcalfeInt8** [comment]
Hi @emanuelpinheiro,

Thanks for the suggestion! I'll keep my eye out for open source AI models that are able to achieve something like that. I've looked a bit into (but haven't tried yet) some models that can perform voice separation (given a track with multiple speakers, separate them into a track for each unique speaker). I wonder if these kinds of models would also work for vocals... 

anyway thanks! I'll keep you posted.

Ryan
